app/src/main/java/skills/future/planer/ui/habit/HabitExtendedTotalAdapter.java

```diff
@@ -64,10 +64,7 @@ private View createViewOfItem(@NonNull ViewGroup parent, int layoutType) {
         AnimateView.singleAnimation(itemView.findViewById(R.id.circularProgressIndicatorHabitDay), context, R.anim.scalezoom2);
         AnimateView.singleAnimation(itemView.findViewById(R.id.circularProgressIndicatorHabitDay), context, R.anim.scalezoom2);
 
-        editButton.setOnClickListener(e -> {
```

Comment:
Dlaczego to zniknęło?

Reply:
Naprawa merga od @M-Szymczyk, listener jest w innym miejscu

Reply:
okk